tests/acceptance/conditions/test_conditions.py

```diff
@@ -97,6 +97,7 @@ def test_user_address_context_variable_verification(testerchain, valid_user_addr
         _recover_user_address(**invalid_signature_context)
 
 
+@pytest.mark.skip(reason="TODO")
```

Comment:
Is there a common theme for the issues encountered in this test file and necessitate skipping the tests?